requests_toolbelt/adapters/appengine.py

```diff
@@ -44,6 +53,75 @@ def __init__(self, validate_certificate=True, *args, **kwargs):
         super(AppEngineAdapter, self).__init__(self, *args, **kwargs)
```

Comment:
I'm curious, why does it need the check? Is this how we would check that https://github.com/shazow/urllib3/pull/763 had been merged properly into requests library?